scripts/tasks/babel.ts

```diff
@@ -0,0 +1,45 @@
+import { transformAsync } from '@babel/core';
+import * as glob from 'glob';
+import fs from 'fs';
+import path from 'path';
+import { EOL } from 'os';
+
+const OEL_REGEX = new RegExp(EOL, 'g');
+
+function addSourceMappingUrl(code: string, loc: string): string {
+  return code + '\n//# sourceMappingURL=' + path.basename(loc);
+}
+
+export async function babel() {
+  const files = glob.sync('{lib,lib-commonjs}/**/*.js');
```

Comment:
Probably need to do lib-amd too (it's only built with `--production`):
```suggestion
  const files = glob.sync('{lib,lib-commonjs,lib-amd}/**/*.js');
```

Reply:
Current implementation i.e. Babel plugin does not support AMD. I don't think that we ever discussed that converged components should have AMD output at all, actually...

Reply:
Hmm good point. It might be good to clarify somehow that this task is only meant to be used for converged. Maybe with naming, or make it throw an error if the current package is missing a relevant dependency?

Reply:
As I went with a config (`.babelrc.json`) technically it's not dependent on converged components or not, it is dependent on a config file 🙄